app/_hub/kong-inc/upstream-oauth/overview/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+The Upstream OAuth plugin lets you manage OAuth flows between {{site.base_gateway}} and an upstream service.
```

Comment:
```suggestion
The Upstream OAuth plugin allows you to manage OAuth flows between {{site.base_gateway}} and an upstream service.
```
This could be me not understanding auth plugins well, but why would I use this one instead of a different auth plugin? Do those other ones not allow you to manage OAuth with an upstream service? How would I know I need this one and not the other ones?

Reply:
It's not you, this is poorly phrased. I'll rephrase.